locallib.php

```diff
@@ -1083,6 +1085,19 @@ function convertsecondstostring($secs) {
     return $hours.$minutes.':'.$seconds;
 }
 
+/**
+ * Gets number of attempts remaining for the user.
+ *
+ * @param int $time timestamp in Unix epoch time
+ * @return string $timestamp timestamp in Europe/Helsinki timezone format dd.mm.yyyy hh.mm:ss
+ */
+function timestampformatter($time) {
+    date_default_timezone_set('Europe/Helsinki');
+    $timestamp = date("H.i:s d.m.Y", $time);
```

Comment:
```suggestion
    $timestamp = date("d.m.Y H.i:s", $time);
```
I think that showing the date first is more user friendly way to show datetime.

Reply:
I think so too. Originally this was the other way around but we changed it after feedback. I'll change it back again.